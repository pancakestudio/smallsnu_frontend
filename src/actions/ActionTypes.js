export const BUILDING_CLICK = "BUILDING_CLICK";
