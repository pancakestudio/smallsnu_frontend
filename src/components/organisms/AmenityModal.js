import React from 'react'
import { Card, ListGroup, Modal, Button } from 'react-bootstrap'
import { FaAngleLeft } from 'react-icons/fa'
import { ModalSpinner } from '../molecules/ModalSpinner'
import { historyPush } from '../../utils/Functions'
import './AmenityModal.css'

export const AmenityModal = ({match, res, cafe, conv, bank, atm}) => {
  let data
  switch(match.params.amenity){
    case "restaurant":
      data = res
      break
    case "cafe":
      data = cafe
      break
    case "conv":
      data = conv
      break
    case "bank":
      data = bank
      break
    case "atm":
      data = atm
      break
    default:
  }

  const handleBack = () => {
    if(data && data.building){
      let bldgNo = data.building.code
      historyPush(`/building/${bldgNo}`)
    } else {
      historyPush('/')
    }
  }

  if(Object.keys(data).length>0){
    return (
      <Modal
        show = {true}
        onHide = {()=>{historyPush('/')}}
        dialogClassName = "amenityModal"
        scrollable = {true}
        centered
      >
        <Modal.Header closeButton>
          <Button className="back" onClick={handleBack}><FaAngleLeft /></Button>
          <Modal.Title className="title">{data.kr_name}</Modal.Title>
        </Modal.Header>
        <Modal.Body className="modalBody">
          <Card className="op_hours border-0">
            <ListGroup variant="flush">
              <ListGroup.Item className="border-left-0 border-right-0 border-top-0"><strong>위치:</strong> {data.building.kr_name}</ListGroup.Item>
              <ListGroup.Item className="border-left-0 border-right-0 border-bottom-0">
                <strong>운영 시간:</strong><br/>
                {data.operating_hours.split('\n').map(line=>(
                  <span key={line}>{line}<br/></span>
                ))}
              </ListGroup.Item>
            </ListGroup>
          </Card>
        </Modal.Body>
      </Modal>
    )
  } else {
    return (
      <ModalSpinner />
    )
  }
}
