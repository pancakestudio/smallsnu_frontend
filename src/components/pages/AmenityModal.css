.amenityModal {
  min-width: 50vw;
}

.amenityModal .title {
  margin: auto 0px;
}
