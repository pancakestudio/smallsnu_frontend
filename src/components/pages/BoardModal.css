.boardModal {
  min-width: 80vw;
}

.boardModal .title{
  margin: auto 0px;
}

.boardModal .center {
  display: flex;
  justify-content: center;
}

.boardModal .pagination {
  margin: auto;
}

.boardModal .noPosts {
  text-align: center;
}
