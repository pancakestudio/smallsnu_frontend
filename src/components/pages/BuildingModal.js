import React from 'react'
import { Modal } from 'react-bootstrap'
import { BuildingInfo } from '../molecules/BuildingInfo'
import { ModalSpinner } from '../molecules/ModalSpinner'
import { historyPush } from '../../utils/Functions'
import './BuildingModal.css'

export const BuildingModal = ({bldg}) => {
  if(bldg && Object.keys(bldg).length>0){
  let info
  if(bldg.info){
    info = bldg.info.split('\n').map(line =>(
      <span key={line}>{line}<br/></span>
    ))
  }
    return(
      <Modal
        show = {true}
        onHide = {()=>{historyPush('/')}}
        dialogClassName = "buildingModal"
        scrollable = {true}
        centered
      >
        <Modal.Header closeButton>
          <Modal.Title>{bldg.krName}</Modal.Title>
        </Modal.Header>

        <Modal.Body>
          <h5>건물 정보</h5>
          <p> {info} </p>
          <BuildingInfo
            bldgNo={bldg.bldgNo}
            rests={bldg.restaurants}
            semis={bldg.seminars}
            posts={bldg.posts}
          />
        </Modal.Body>

      </Modal>
    )
  } else {
    return (
      <ModalSpinner/>
    )
  }
}
