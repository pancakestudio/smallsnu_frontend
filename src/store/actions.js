import * as types from './actionTypes'

/******** Map ********/

export function requestBuilding(bldgNo){
  return{
    type: types.REQUEST_BUILDING,
    bldgNo
  };
}

export function getBuildingSuccess(krName, bldgNo, info, restaurants, seminars, lectures, posts){
  return{
    type: types.GET_BUILDING_SUCCESS,
    krName,
    bldgNo,
    info,
    restaurants,
    seminars,
    lectures,
    posts,
  }
}

export function getBuildingFailure(error){
  return{
    type: types.GET_BUILDING_FAILURE,
    error
  }
}

export function zoomChanged(zoomLevel){
  return {
    type: types.ZOOM_CHANGED,
    zoomLevel
  }
}

export function hideMarkers(){
  return {
    type: types.HIDE_MARKERS
  }
}

/******** SideBar ********/

export function toggleSidebar(){
  return {
    type: types.TOGGLE_SIDE_BAR
  }
}

export function toggleShuttleMenuMarker(){
  return{
    type: types.TOGGLE_SHUTTLE_MENU_MARKER
  }
}

/******** SearchBar ********/

export function search(bldgNo, bldgPos){
  return{
    type: types.SEARCH,
    bldgNo,
    bldgPos
  }
}

/******** Shuttle ********/
export function requestAllShuttles(){
  return{
    type: types.REQUEST_ALL_SHUTTLE
  }
}

export function getAllShuttleSuccess(stations){
  return{
    type: types.GET_ALL_SHUTTLE_SUCCESS,
    stations
  }
}

export function toggleShuttleMarker(){
  return{
    type: types.TOGGLE_SHUTTLE_MARKER
  }
}

export function requestAllRevShuttles(){
  return{
    type: types.REQUEST_ALL_REV_SHUTTLE
  }
}

export function getAllRevShuttleSuccess(stations){
  return{
    type: types.GET_ALL_REV_SHUTTLE_SUCCESS,
    stations
  }
}

export function toggleRevShuttleMarker(){
  return{
    type: types.TOGGLE_REV_SHUTTLE_MARKER
  }
}

/******** Post ********/

export function requestBoard(bldgNo){
  return {
    type: types.REQUEST_BOARD,
    bldgNo
  }
}

export function getBoardSuccess(posts, bldgNo){
  return {
    type: types.GET_BOARD_SUCCESS,
    posts,
    bldgNo
  }
}

export function getBoardFailure(error){
  return {
    type: types.GET_BOARD_FAILURE,
    error
  }
}

export function requestPost(id) {
  return {
    type: types.REQUEST_POST,
    id
  }
}

export function getPostSuccess(post){
  return {
    type: types.GET_POST_SUCCESS,
    post
  }
}

export function getPostFailure(error){
  return {
    type: types.GET_POST_FAILURE,
    error
  }
}

export function savePost(post, bldgNo){
  return{
    type: types.SAVE_POST,
    post,
    bldgNo
  }
}

export function savePostSuccess(bldgNo){
  return {
    type: types.SAVE_POST_SUCCESS,
    bldgNo
  }
}

export function savePostFailure(error){
  return {
    type: types.SAVE_POST_FAILURE,
    error
  }
}

export function editPost(post, bldgNo){
  return{
    type: types.EDIT_POST,
    post,
    bldgNo,
  }
}

export function editPostSuccess(postId){
  return {
    type: types.EDIT_POST_SUCCESS,
    postId
  }
}

export function editPostFailure(error){
  return {
    type: types.EDIT_POST_FAILURE,
    error
  }
}

export function editPostFlag(){
  return{
    type: types.EDIT_POST_FLAG,
  }
}

export function deletePost(post, bldgNo, password){
  return{
    type: types.DELETE_POST,
    post,
    bldgNo,
    password
  }
}

export function deletePostSuccess(bldgNo){
  return {
    type: types.DELETE_POST_SUCCESS,
    bldgNo
  }
}

export function deletePostFailure(error){
  return {
    type: types.DELETE_POST_FAILURE,
    error
  }
}

export function showWritePost(bldgNo) {
  return{
    type: types.SHOW_WRITE_POST,
    bldgNo
  };
}

export function hideWritePost() {
  return {
    type: types.HIDE_WRITE_POST,
  }
}

export function showPasswordCheck(target){
  return{
    type: types.SHOW_PASSWORD_CHECK,
    target
  }
}

export function hidePasswordCheck(){
  return{
    type: types.HIDE_PASSWORD_CHECK
  }
}

export function postLike(postId) {
  return {
    type: types.POST_LIKE,
    postId
  }
}

export function postLikeSuccess(postId) {
  return {
    type: types.POST_LIKE_SUCCESS,
    postId
  }
}

export function postLikeFailure(error) {
  return {
    type: types.POST_LIKE_FAILURE,
    error
  }
}

export function saveComment(comment, postId){
  return {
    type: types.SAVE_COMMENT,
    comment,
    postId
  }
}

export function saveCommentSuccess(postId){
  return {
    type: types.SAVE_COMMENT_SUCCESS,
    postId
  }
}

export function saveCommentFailure(error){
  return {
    type: types.SAVE_COMMENT_FAILURE,
    error
  }
}

export function showEditComment(comment){
  return {
    type: types.SHOW_EDIT_COMMENT,
    comment
  }
}

export function hideEditComment(){
  return {
    type: types.HIDE_EDIT_COMMENT
  }
}

export function editComment(comment, postId){
  return {
    type: types.EDIT_COMMENT,
    comment,
    postId
  }
}

export function editCommentSuccess(postId){
  return {
    type: types.EDIT_COMMENT_SUCCESS,
    postId
  }
}

export function editCommentFailure(error){
  return {
    type: types.EDIT_COMMENT_FAILURE,
    error
  }
}

export function deleteComment(comment, postId, password){
  return {
    type: types.DELETE_COMMENT,
    comment,
    postId,
    password
  }
}

export function deleteCommentSuccess(msg){
  return {
    type: types.DELETE_COMMENT_SUCCESS,
    msg
  }
}

export function deleteCommentFailure(error){
  return {
    type: types.DELETE_COMMENT_FAILURE,
    error
  }
}

export function commentLike(commentId, postId) {
  return {
    type: types.COMMENT_LIKE,
    commentId,
    postId
  }
}

export function commentLikeSuccess(postId) {
  return {
    type: types.COMMENT_LIKE_SUCCESS,
    postId
  }
}

export function commentLikeFailure(error) {
  return {
    type: types.COMMENT_LIKE_FAILURE,
    error
  }
}

export function changeBoardPage(page){
  return {
    type: types.CHANGE_BOARD_PAGE,
    page
  }
}

export function wrongPassword(){
  return {
    type: types.WRONG_PASSWORD,
  }
}

/******** Restaurant ********/

export function toggleResMarker(){
  return{
    type: types.TOGGLE_RES_MARKER,
  }
}

export function requestRestaurant(id){
  return{
    type: types.REQUEST_RESTAURANT,
    id
  }
}

export function getRestaurantSuccess(restaurant){
  return{
    type : types.GET_RESTAURANT_SUCCESS,
    restaurant,
  }
}

export function getRestaurantFailure(error){
  return{
    type: types.GET_RESTAURANT_FAILURE,
    error
  }
}

export function requestAllRestaurants(){
  return{
    type: types.REQUEST_ALL_RESTAURANTS,
  }
}


export function getAllRestaurantsSuccess(restaurants){
  return{
    type : types.GET_ALL_RESTAURANTS_SUCCESS,
    restaurants,
  }
}

export function getAllRestaurantsFailure(error){
  return{
    type: types.GET_ALL_RESTAURANTS_FAILURE,
    error
  }
}

/******** Seminar ********/

export function toggleSemiMarker(){
  return{
    type: types.TOGGLE_SEMI_MARKER,
  }
}

export function changeSeminarPage(page){
  return {
    type: types.CHANGE_SEMINAR_PAGE,
    page
  }
}

export function requestSeminar(id){
  return {
    type: types.REQUEST_SEMINAR,
    id
  }
}

export function getSeminarSuccess(seminar){
  return {
    type: types.GET_SEMINAR_SUCCESS,
    seminar
  }
}

export function getSeminarFailure(error){
  return {
    type: types.GET_SEMINAR_FAILURE,
    error
  }
}

export function requestBldgSeminars(bldgNo){
  return {
    type: types.REQUEST_BLDG_SEMINARS,
    bldgNo
  }
}

export function getBldgSeminarsSuccess(seminars, bldgNo){
  return {
    type: types.GET_BLDG_SEMINARS_SUCCESS,
    seminars,
    bldgNo
  }
}

export function getBldgSeminarsFailure(error){
  return {
    type: types.GET_BLDG_SEMINARS_FAILURE,
    error
  }
}

export function requestAllSeminars(){
  return {
    type: types.REQUEST_ALL_SEMINARS,
  }
}

export function getAllSeminarsSuccess(seminars){
  return {
    type: types.GET_ALL_SEMINARS_SUCCESS,
    seminars
  }
}

export function getAllSeminarsFailure(error){
  return {
    type: types.GET_ALL_SEMINARS_FAILURE,
    error
  }
}

/******** Bank ********/

export function toggleBankMarker(){
  return{
    type: types.TOGGLE_BANK_MARKER,
  }
}

export function requestAllBanks(){
  return{
    type: types.REQUEST_ALL_BANKS,
  }
}

export function getAllBanksSuccess(banks){
  return{
    type: types.GET_ALL_BANKS_SUCCESS,
    banks,
  }
}

export function getAllBanksFailure(error){
  return{
    type: types.GET_ALL_BANKS_FAILURE,
    error,
  }
}

export function requestBank(id){
  return{
    type: types.REQUEST_BANK,
    id
  }
}

export function getBankSuccess(bank){
  return{
    type: types.GET_BANK_SUCCESS,
    bank
  }
}

export function getBankFailure(error){
  return{
    type: types.GET_BANK_FAILURE,
    error
  }
}


/******** ATM ********/

export function toggleATMMarker(){
  return{
    type: types.TOGGLE_ATM_MARKER,
  }
}

export function requestAllATMs(){
  return{
    type: types.REQUEST_ALL_ATMS,
  }
}

export function getAllATMsSuccess(atms){
  return{
    type: types.GET_ALL_ATMS_SUCCESS,
    atms,
  }
}

export function getAllATMsFailure(error){
  return{
    type: types.GET_ALL_ATMS_FAILURE,
    error,
  }
}

export function requestATM(id){
  return{
    type: types.REQUEST_ATM,
    id
  }
}

export function getATMSuccess(atm){
  return{
    type: types.GET_ATM_SUCCESS,
    atm
  }
}

export function getATMFailure(error){
  return{
    type: types.GET_ATM_FAILURE,
    error
  }
}

/******** Cafe ********/

export function toggleCafeMarker(){
  return{
    type: types.TOGGLE_CAFE_MARKER,
  }
}

export function requestAllCafes(){
  return{
    type: types.REQUEST_ALL_CAFES,
  }
}

export function getAllCafesSuccess(cafes){
  return{
    type: types.GET_ALL_CAFES_SUCCESS,
    cafes,
  }
}

export function getAllCafesFailure(error){
  return{
    type: types.GET_ALL_CAFES_FAILURE,
    error,
  }
}

export function requestCafe(id){
  return{
    type: types.REQUEST_CAFE,
    id
  }
}

export function getCafeSuccess(cafe){
  return{
    type: types.GET_CAFE_SUCCESS,
    cafe
  }
}

export function getCafeFailure(error){
  return{
    type: types.GET_CAFE_FAILURE,
    error
  }
}

/******** Conv ********/

export function toggleConvMarker(){
  return{
    type: types.TOGGLE_CONV_MARKER,
  }
}

export function requestAllConves(){
  return{
    type: types.REQUEST_ALL_CONVES,
  }
}

export function getAllConvesSuccess(conves){
  return{
    type: types.GET_ALL_CONVES_SUCCESS,
    conves,
  }
}

export function getAllConvesFailure(error){
  return{
    type: types.GET_ALL_CONVES_FAILURE,
    error,
  }
}

export function requestConv(id){
  return{
    type: types.REQUEST_CONV,
    id
  }
}

export function getConvSuccess(conv){
  return{
    type: types.GET_CONV_SUCCESS,
    conv
  }
}

export function getConvFailure(error){
  return{
    type: types.GET_CONV_FAILURE,
    error
  }
}


/******** Main ********/

export function refreshData(){
  return {
    type: types.REFRESH_DATA
  }
}
