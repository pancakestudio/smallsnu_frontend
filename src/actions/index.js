import * as types from './ActionTypes';

export function buildingClick(e){
  return{
    type: types.BUILDING_CLICK,
    e
  };
}
