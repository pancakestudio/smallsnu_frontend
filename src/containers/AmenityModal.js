import { connect } from 'react-redux'
import { AmenityModal } from '../components/pages/AmenityModal'

const mapStateToProps = (state) => {
  return{
    res : state.selectedRes,
    cafe: state.selectedCafe,
    conv: state.selectedConv,
    bank: state.selectedBank,
    atm: state.selectedATM,
  }
}

export default connect(mapStateToProps, null)(AmenityModal)
