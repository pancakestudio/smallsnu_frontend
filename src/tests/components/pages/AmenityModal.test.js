import React from 'react'
import { Provider } from 'react-redux'
import { AmenityModal } from '../../../components/pages/AmenityModal'
import ConnectedAmenityModal from '../../../containers/AmenityModal'
import configureStore from 'redux-mock-store'
import { shallow, mount } from 'enzyme'
import * as actions from '../../../store/actions'


describe('AmenityModal', () => {
  global.window = { location: { pathname: null } };
  let component

  it('renders correctly', () => {
    component = shallow(
    <AmenityModal
        match = {{params: {amenity: "restaurant", id: "1"}}}
        res = {{
          kr_name:'학생회관 식당',
          building: {kr_name: "학생회관", code: "62"},
          operating_hours:"오전 10시 - 오후 6시",
          location: '학생회관 1층'
        }}
        show = {true}
    />)
  })

  it('matches snapshot', () => {
    expect(component).toMatchSnapshot()
  })

  it('has a modal', ()=>{
    expect(component.find('Bootstrap(Modal)').exists()).toBe(true)
  })

  it('has a correct restaurant name', ()=>{
    expect(component.find('ModalTitle').text()).toBe('학생회관 식당')
  })

  it('has a correct building name', ()=>{
    expect(component.find('Bootstrap(ListGroupItem)').at(0).text()).toBe('위치: 학생회관 1층')
  })

  it('has a correct operating hours', ()=>{
    expect(component.find('Bootstrap(ListGroupItem)').at(1).text()).toBe('운영 시간: 오전 10시 - 오후 6시')
  })

  it('has a correct cafe name', ()=>{
    component.setProps({match: {params: {amenity: "cafe", id: "1"}}})
    component.setProps({cafe: {
      kr_name: '학생회관 카페',
      building: {kr_name: "학생회관", code: '62'},
      operating_hours:"오전 10시 - 오후 6시",
      location: '학생회관 2층'
    }})
    expect(component.find('ModalTitle').text()).toBe('학생회관 카페')
  })

  it('has a correct building name', ()=>{
    expect(component.find('Bootstrap(ListGroupItem)').at(0).text()).toBe('위치: 학생회관 2층')
  })

  it('has a correct operating hours', ()=>{
    expect(component.find('Bootstrap(ListGroupItem)').at(1).text()).toBe('운영 시간: 오전 10시 - 오후 6시')
  })

  it('has a correct conv name', ()=>{
    component.setProps({match: {params: {amenity: "conv", id: "1"}}})
    component.setProps({conv: {
      kr_name: '학생회관 편의점',
      building: {kr_name: "학생회관", code: '62'},
      operating_hours: "오전 10시 - 오후 10시",
    }})
    expect(component.find('ModalTitle').text()).toBe('학생회관 편의점')
  })

  it('has a correct building name', ()=>{
    expect(component.find('Bootstrap(ListGroupItem)').at(0).text()).toBe('위치: 학생회관')
  })

  it('has a correct operating hours', ()=>{
    expect(component.find('Bootstrap(ListGroupItem)').at(1).text()).toBe('운영 시간: 오전 10시 - 오후 10시')
  })

  it('has a correct bank name', ()=>{
    component.setProps({match: {params: {amenity: "bank", id: "1"}}})
    component.setProps({bank: {
      kr_name: '학생회관 은행',
      building: {kr_name: "학생회관", code: '62'},
      operating_hours: "오전 10시 - 오후 4시",
      location: '학생회관 1층'
    }})
    expect(component.find('ModalTitle').text()).toBe('학생회관 은행')
  })

  it('has a correct building name', ()=>{
    expect(component.find('Bootstrap(ListGroupItem)').at(0).text()).toBe('위치: 학생회관 1층')
  })

  it('has a correct operating hours', ()=>{
    expect(component.find('Bootstrap(ListGroupItem)').at(1).text()).toBe('운영 시간: 오전 10시 - 오후 4시')
  })

  it('has a correct atm name', ()=>{
    component.setProps({match: {params: {amenity: "atm", id: "1"}}})
    component.setProps({atm: {
      kr_name: '학생회관 ATM',
      building: {kr_name: "학생회관", code: '62'},
      operating_hours: "오전 10시 - 오후 9시",
      location: '학생회관 1층'
    }})
    expect(component.find('ModalTitle').text()).toBe('학생회관 ATM')
  })

  it('has a correct building name', ()=>{
    expect(component.find('Bootstrap(ListGroupItem)').at(0).text()).toBe('위치: 학생회관 1층')
  })

  it('has a correct operating hours', ()=>{
    expect(component.find('Bootstrap(ListGroupItem)').at(1).text()).toBe('운영 시간: 오전 10시 - 오후 9시')
  })

  it('handles back correctly', ()=>{
    component.find('.back').simulate('click')
    expect(global.window.location.pathname).toEqual('/building/62')

    component.setProps({atm: {
      kr_name: '학생회관 ATM',
      operating_hours: "오전 10시 - 오후 9시",
      location: '학생회관 1층'
    }})
    component.find('.back').simulate('click')
    expect(global.window.location.pathname).toEqual('/')
  })

  it('calls functions', ()=>{
    component.find('Bootstrap(Modal)').simulate('hide')
    expect(global.window.location.pathname).toEqual('/')
  })
})

describe('ConnectedAmenityModal', () => {
  const initialState = {
    selectedRes: {
      kr_name:'학생회관 식당',
      building: {kr_name: "학생회관", code: "62"},
      operating_hours:"오전 10시 - 오후 6시",
      location: '학생회관 1층'
    }
  }
  const mockStore = configureStore()
  let store, component

  it('renders correctly', () => {
    store = mockStore(initialState)
    component = mount(
      <Provider store={store}>
        <ConnectedAmenityModal match={{params: {amenity: "restaurant", id: "1"}}}/>
      </Provider>
    )
  })

  it('matches snapshot', () => {
    expect(component).toMatchSnapshot()
  })
})
