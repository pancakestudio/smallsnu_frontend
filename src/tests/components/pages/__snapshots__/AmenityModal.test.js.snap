// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AmenityModal matches snapshot 1`] = `ShallowWrapper {}`;

exports[`ConnectedAmenityModal matches snapshot 1`] = `ReactWrapper {}`;
